roles/kubernetes/master/templates/kube-scheduler-policy.yaml.j2

```diff
@@ -0,0 +1,18 @@
+{
+"kind" : "Policy",
+"apiVersion" : "v1",
+"predicates" : [
```

Comment:
It feels like these are things that would be worthwhile to abstract into a template, but I'm not super familiar with the use case.

Reply:
I am not sure if there are of any use outside Openstack deployment scenario. In Openstack you may have different AZ setup for compute and storage, not sure if there is any other cloud with the same behavior.
And usually you don't want to disable default scheduler restrictions, do you?